### DISCUSSION TITLE
Twitter Space: OpenAI's Whisper

### DISCUSSION THREAD

**Discussion by BekahHW** [deepgram/community#29] [General help]
## OpenAI's Whisper Twitter Space Follow-up

If you have questions, comments, or thoughts on OpenAI's Whisper and we didn't get to them in the space, drop them here!

![Twitter Space image card with date Sept 23 at 10a PT with Bekah and Nick Martin's headshots](https://user-images.githubusercontent.com/34313413/192002401-9005e6ab-a8e5-44de-91c2-e545c4b19b77.png)

**Comment by dotAadarsh** [comment]
Short and informative space! Loved it! So I have built a mini [app](https://dotaadarsh-streamlit-projects-app-6djlzp.streamlitapp.com/whisper) that uses the [Deepgram OpenAI Whisper Speech Recognition API](https://blog.deepgram.com/use-openai-whisper-speech-recognition-with-the-deepgram-api/) to transcribe some of the voices of Apollo mission and It's really good. Check it out [↗️](https://dotaadarsh-streamlit-projects-app-6djlzp.streamlitapp.com/whisper) and I am excited about more twitter spaces.
Thank You!

**Reply by michaeljolley** [reply]
Nice!

**Reply by natalierutgers** [reply]
This is awesome!
